app/assets/javascripts/past_teams.js

```diff
@@ -0,0 +1,16 @@
+$(function() {
+  var $selectedYear = $('select#past_teams');
```

Comment:
Yes, you're right, it's used by this type of input element: [here's the code for it](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/inputs/city_input.rb)

It's not 100% the same, but the usage in views is similar in syntax.